integrations/extensions/docs/elasticsearch-install-and-setup/how_to_install_docker.md

```diff
@@ -0,0 +1,141 @@
+
+<img src="./assets/podman_machine_running.png" width="300" height="100"/>
+
+However, it is recommended to update the configurations of the machine by :
+
+* clicking on the edit button
+* choosing a reasonable CPU, Memory and Disk Size 
+* click on "Update"
+
+<img src="./assets/podman_machine_edit.png" width="300" height="200"/>
+
+This will restart the machine with the updated configurations.
+
+### Install Docker and Compose Extensions
+
+Under Settings > Extensions, click on the "install" icon for the Docker and Compose extensions. Follow the prompts on the screen.
+
+<img src="./assets/podman_extensions.png" width="300" height="200"/>
+
+### Emulating Docker CLI 
+
+In order to run `docker` commands using podman, you would have to follow steps as listed in the Step 1 and 3 of the [documentation](https://podman-desktop.io/docs/migrating-from-docker/emulating-docker-cli-with-podman#procedure)
+
+This creates a proxy to podman using the `docker` command on the interface and podman behind the scenes
+
+### Verifying the install
+
+You can now run the below on your terminal or console to verify the setup is working.
+
+```bash
+docker run hello-world
+```
+
+#### Useful Links:
+
+* [https://thenewstack.io/install-and-use-podman-desktop-gui-to-manage-containers/](https://thenewstack.io/install-and-use-podman-desktop-gui-to-manage-containers/)
+* [https://developers.redhat.com/articles/2023/09/27/how-install-and-use-podman-desktop-windows#getting_started_with_podman_desktop](https://developers.redhat.com/articles/2023/09/27/how-install-and-use-podman-desktop-windows#getting_started_with_podman_desktop)
+
+## Option 4: Colima
+
+**IMPORTANT NOTE**: This approach requires using a CLI(Command Line Interface) and does not have a button-click approach. This is recommended for users who prefer a CLI-focused approach to Docker Desktop's GUI focus.
+
+[Colima](https://github.com/abiosoft/colima) is a new open-source project that bundles the container management tool [lima](https://github.com/lima-vm/lima) with a docker (linux) back-end. This is similar to what Docker Desktop actually does, but Colima and Lima are entirely open-source and just focused on running containers. They work on both `amd64` and `arm64` (M1) macs. Colima does not require installation of Docker Desktop, or does it require paying a licensing fee to Docker, Inc.
+
+
+### For MacOS and Linux
+
+1. First, install `homebrew` package manager by running the below command on a shell:
+	
+	```shell
+	/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+	```
+
+2. Next, install `colima` , `docker` and `compose` using homebrew 
+
+	```shell
+	brew install colima
+	brew install docker docker-compose
+	```
+3. 	Start `colima` with the following command. This may take a while the first time, so wait for it to fully complete
+
+	```shell
+	colima start
```

Comment:
For web crawling (to be able to run Enterprise Search and Kibana at the same time), I needed to do the following:

```
colima start --cpu 4 --memory 6
```

I think it would be good to provide this as an explicit example.